### DISCUSSION TITLE
Regarding Search option in menu bar

### DISCUSSION THREAD

**Discussion by theonlyanson** [adityatelange/hugo-PaperMod#1405] [General]
I had added a search option by referring the documentation. Initially it worked, but after some time it stopped working. Now Search option is properly visible but When I write into it, I'm not able to search anything.

Page link: https://theonlyanson.github.io/

**Comment by RoneoOrg** [comment]
The browser console shows a 404 error for https://theonlyanson.github.io/index.json

**Reply by theonlyanson** [reply]
What am I supposed to do now?
Could u give me the steps to solve this??

**Reply by RoneoOrg** [reply]
First, make sure JSON is generated by Hugo config file:

```
[outputs]
	home = ["HTML","RSS","JSON"]
```

Or in YAML format:

```
outputs:
  home:
    - HTML
    - RSS
    - JSON
```

**Reply by theonlyanson** [reply]
![Screenshot from 2024-01-27 08-25-41](https://github.com/adityatelange/hugo-PaperMod/assets/62369900/671626b9-eb52-468c-8828-77a8391e119b)
Yes, it's generated

**Reply by RoneoOrg** [reply]
Indeed you have a template, but check the config file too^

**Reply by theonlyanson** [reply]
But How to check that?? could u guide me plzzz...
